R/interface.R

```diff
@@ -289,9 +289,16 @@ sourceExamples <- function(file){
 rcppoctave <- function(..., plugin = 'RcppOctave'){
 	if( !requireNamespace('inline') )
 		stop("Package 'inline' is required to define RcppOctave functions.")
-	#qlibrary(inline)
-	#rcpp(..., plugin = plugin)
-	cxxfunction(..., plugin = plugin)
+	#qlibrary('inline')
```

Comment:
Here `qlibrary` is from `pkgmaker`, which is loaded anyway.
This is to force attaching `inline` quietly, but only when necessary, this is why `inline` is only in Suggests.